doc/examples/filters/plot_enhancement_measure.py

```diff
@@ -0,0 +1,60 @@
+"""
+=========================================
+Quantitative Measure of Image Enhancement
+=========================================
+In this example, we will quantify the effect of applying contrast enhancement
+to an image using a measure called EME [1]_.
+
+It is defined as the log of the ratio of local maximum to minimum within a
+sliding window of given size. This measure aims to provide a perceptual
+estimate of image quality, useful when comparing image enhancement algorithms.
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage import data
+from skimage.filters import gaussian
+from skimage.exposure import equalize_adapthist
+from skimage.metrics import simple_metrics
+
+img_gray = data.camera()
+img_rgb = data.astronaut()
+
+
+def compare_side_by_side(before, after):
+    fig, ax = plt.subplots(nrows=1, ncols=2, figsize=(16, 14))
```

Comment:
```suggestion
    fig, ax = plt.subplots(nrows=1, ncols=2)
```
I think removing the forced figure size leads to a nicer layout. Otherwise the images seem very small.